jenkins/jenkins.yaml

```diff
@@ -0,0 +1,58 @@
+---
+# Source: roadie-kubernetes-cluster-access/templates/cluster-role.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I think we probably don't need this stuff at all - we just want a broker running in the cluster. I think it shouldn't need access to the cluster apis like the k8s one does as the k8s one is fetching all sorts of info from the control plane